Makefile

```
@@ -18,13 +18,20 @@ run: build-mainnet-release
 migrate-testnet: build-testnet-release
 	@${BIN} migrate --account ${ACCOUNT_ID} --key ${ACCOUNT_KEY}  --file contract_state.borsh
 
-build-index:
+migrate-testnet-indexed: build-testnet-release
+	@${BIN} migrate --account ${ACCOUNT_ID} --key ${ACCOUNT_KEY}  --file for-migtation.borsh
```

Needless whitespace and typo in `for-migration.borsh`

```suggestion
	@${BIN} migrate --account ${ACCOUNT_ID} --key ${ACCOUNT_KEY} --file for-migration.borsh
```